src/app/files/files-edit/files-edit.component.html

```diff
@@ -3,57 +3,57 @@
 <grid-main [class]="'width:100%;'" [centered]="true">
```

Comment:
are you adding this component back? It doesnt exist

Reply:
What do you mean? It exists :) I think I just saved this one and it was formatted, I haven't changed anything.

Reply:
Ohh it does exist lol nevermind.